docs-conceptual/azureadps-2.0-preview/overview.md

```diff
@@ -10,6 +10,9 @@ ms.custom: posh-docs-conceptual
 ---
 # Azure Active Directory PowerShell for Graph
 
+>[!IMPORTANT]
+> Azure AD PowerShell is planned for deprecation. See the [deprecation update](https://techcommunity.microsoft.com/t5/azure-active-directory-identity/azure-ad-change-management-simplified/ba-p/2967456) for more details on the deprecation plans. We recommend that you start trying [Microsoft Graph PowerShell](/powershell/microsoftgraph/overview). Microsoft Graph PowerShell is the replacement for Azure AD PowerShell and is recommended for interacting with Azure AD. See the [migration FAQ](../azureadps-2.0/migration-faq.yml) for answers to frequent migration queries.
```

Comment:
Similar comment. Remove the passive voice and provide a more compelling description on "why"